src/components/DeleteVehicle.js

```diff
@@ -25,8 +25,8 @@ export default function DeleteVehicle() {
       const response = await axios.delete(`http://localhost:3000/api/v1/items/${itemId}`);
       setMessage(response.data.message);
     } catch (error) {
-      setMessage('Error deleting item');
-      console.error('Error deleting item:', error);
+      setMessage('Error deleting the vehicle');
+      throw ('Error deleting the vehicle:', error);
     }
   };
 
```

Comment:
The error messege should be descriptive like "Cannot delete Vehicle, As you have reservation for this Vehicle."